### DISCUSSION TITLE
Allow more than one cable lift on giga coaster

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16620] [Feature requests]
[Github giga.zip](https://github.com/OpenRCT2/OpenRCT2/files/8037151/Github.giga.zip)
when you have more than one station on a giga coaster, the game blocks you from placing a second cable lift. In the save file, edit the coaster and try to build a cable lift and you will get an error saying only one per ride. I propose allowing more than one cable lift per giga coaster
-

**Comment by duncanspumpkin** [comment]
Cable lifts are really quite special in how they work. This would require quite an extensive rework to make it work. Are there any examples in the real world where a double cable lift exists?

**Comment by ocalhoun6** [comment]
I'm sure it would indeed be a huge headache to code.

But even without any real-world examples, it does seem to make a lot of sense, especially in the case of a mobius coaster using cable lift hills from each synchronized station. It would be a nice option to have.

That said, it would still be a very _niche_ option, and rarely used ... so likely not worth the high amount of effort it would take to code that. (Especially when you could do _almost_ the same thing by just having a pair of separate dueling/racing giga coasters instead of a single mobius.)

But yeah, it would be pretty great. _Ideally_, you'd be able to make _any_ lift hill on the giga coaster into a cable lift hill.

**Reply by 733737** [reply]
I didn't know it would be so difficult, but  if someone wants to code it I'm all for that

**Comment by TheAtomTwister** [comment]
I'm actually curious if the cable lift hill and the station are directly connected.  If they are, I wonder if when new track save formats come along one can add a cable lift base piece that does the same thing as a station.  

So how actually do the cable lifts work?

**Reply by 733737** [reply]
I'd like my pie in the sky request of every coaster in game having cable lift option by 2030

**Reply by TheAtomTwister** [reply]
I have something in the works that'll bring every track element and every train type of every roller coaster there is and that there has ever been to RCT2.  I'm hoping for 2023 though, but I can't make any promises. ;)
